nifi-nar-bundles/nifi-email-bundle/nifi-email-processors/src/main/java/org/apache/nifi/processors/email/ListenSMTP.java

```diff
@@ -106,26 +90,17 @@
             .addValidator(StandardValidators.PORT_VALIDATOR)
             .build();
 
-    protected static final PropertyDescriptor SMTP_HOSTNAME = new PropertyDescriptor.Builder()
-            .name("SMTP_HOSTNAME")
-            .displayName("SMTP hostname")
-            .description("The hostname to be embedded into the banner displayed when an " +
-                    "SMTP client connects to the processor TCP port .")
-            .required(true)
-            .expressionLanguageSupported(false)
-            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
-            .build();
-
-    protected static final PropertyDescriptor SMTP_MAXIMUM_CONNECTIONS = new PropertyDescriptor.Builder()
+    static final PropertyDescriptor SMTP_MAXIMUM_CONNECTIONS = new PropertyDescriptor.Builder()
```

Comment:
@olegz is SMTP_MAXIMUM_CONNECTIONS being enforced at all? 
```

$ date && telnet 0 2525
Fri Aug 12 09:02:31 AEST 2016
Trying 0.0.0.0...
Connected to 0.
Escape character is '^]'.
220 localhost ESMTP Apache NiFi
```

$ date && telnet 0 2525
Fri Aug 12 09:02:30 AEST 2016
Trying 0.0.0.0...
Connected to 0.
Escape character is '^]'.
220 localhost ESMTP Apache NiFi

I did this test and it doesn't seem to be:

```
$ date  && netstat -an| grep 2525 | pcregrep "2525\s+ES"
Fri Aug 12 09:02:33 AEST 2016
tcp        0      0 0.0.0.0:2525                0.0.0.0:*                   LISTEN
tcp        0      0 127.0.0.1:38840             127.0.0.1:2525              ESTABLISHED
tcp        0      0 127.0.0.1:38842             127.0.0.1:2525              ESTABLISHED
```